src/cljs/villagebookUI/components/utils.cljs

```diff
@@ -16,15 +17,21 @@
   [attributes]
   [:input (assoc-target-value attributes :on-change)])
 
-(defn label
-  ([color]
-   [:div.item-label {:style {"backgroundColor" color}}])
-  ([color on-change-handler]
-   [:span
-    [input {:type      "color"
-            :id        "new-item-label"
-            :style     {:display "none"}
-            :value     color
-            :on-change #(on-change-handler %)}]
-    [:label.item-label.new-item-label  {:for   "new-item-label"
-                                        :style {"backgroundColor" color}}]]))
+(defn org-label
```

Comment:
I would debate having an organisation specific name like `org-label` in a `utils` namespace that's not coupled to organisations in any way. Don't think this is a blocker for merge though.